information/licensing/Concurrent_sessions_based_licensing.md

```diff
@@ -0,0 +1,66 @@
+# Concurrent Sessions-Based Licensing
+
+Instances of @@name which are registered and paid by the subscription plans **Small Business Basic** and **Small Business Advanced** are licensed under the **Concurrent Sessions-Based** licensing.
+This type of license allows the client to run a **number of active sessions** at the same time.
+
+## Single Concurrent Session Definition
+
+A **single concurrent session** for an @@name instance is an active connection established between **a single desktop application**, or **web** or **mobile client**, on a single user's **device**, on one side, and the **servers** on which the @@name system's instance and the user database are present and operate, on the other.  
```

Comment:
The note for the servers is unnecessary. Just specify the user actions.

Reply:
Regarding remarks: 

The note for the servers is unnecessary. Just specify the user actions. - The note for the servers is removed. Used "@@name (ERP.net) instance" instead.